bundles/org.openhab.binding.melcloud/src/main/java/org/openhab/binding/melcloud/internal/MelCloudBindingConstants.java

```diff
@@ -42,21 +43,31 @@ public class MelCloudBindingConstants {
     // List of all Channel ids
     public static final String CHANNEL_POWER = "power";
     public static final String CHANNEL_OPERATION_MODE = "operationMode";
+    public static final String CHANNEL_OPERATION_MODE_STRING = "operationModeString";
```

Comment:
Duplicate channel, but just a different representation? I would not expect this channel to be added in that case. What is the use case?